core/database.py

```
@@ -441,10 +437,17 @@ async def remove_access_control_user(user_id: int) -> bool:
-        result = await conn.execute("DELETE FROM access_control_users WHERE user_id = $1", user_id)
-        deleted_count = int(result.split()[-1]) if result else 0
-        return deleted_count > 0
+    try:
+        async with pool.acquire() as conn:
+            result = await conn.execute(
+                "DELETE FROM access_control_users WHERE user_id = $1",
+                user_id,
+            )
+            deleted_count = int(result.split()[-1]) if result else 0
+            return deleted_count > 0
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)